buildSrc/src/main/groovy/org/opensearch/migrations/common/CommonUtils.groovy

```diff
@@ -65,7 +65,8 @@ class CommonUtils {
             }
 
             copyFile("jars", "/jars")
-            def jvmParams = "-XX:MaxRAMPercentage=80.0 -XX:+ExitOnOutOfMemoryError"
+            // Leave memory headroom for native execution which allocates memory outside jvm
+            def jvmParams = "-XX:+UseContainerSupport -XX:MaxRAMPercentage=60.0 -XX:+ExitOnOutOfMemoryError"
```

Comment:
I'm curious what issue you ran into to require or encourage this change?

Reply:
@AndreKurait have you rebased or edited history since I created this PR to remove this change? I didn't add this one